src/screens/AccountEdit.js

```diff
@@ -17,24 +17,24 @@
 'use strict';
 
 import React from 'react';
-import { ScrollView, StyleSheet, Text } from 'react-native';
+import { ScrollView, StyleSheet } from 'react-native';
 import { Subscribe } from 'unstated';
 import colors from '../colors';
-import fonts from '../fonts';
 import AccountCard from '../components/AccountCard';
 import TextInput from '../components/TextInput';
 import AccountsStore from '../stores/AccountsStore';
 
 export default class AccountEdit extends React.PureComponent {
-	static navigationOptions = {
-		title: 'Edit Account'
-	};
-
 	constructor(props) {
 		super(props);
 	}
 
 	render() {
+		const onNameInput = async (accounts, name) => {
```

Comment:
this should be brought out of the  `render()`

Reply:
Good point, I will also check if there is same issue in other component.